### PR TITLE
Fix Global::Pass

### PR DESCRIPTION
**Pull request by kkoopa**

### CODE REVIEW

File: test/cpp/persistent.cpp [hunk 1]
```diff
@@ -49,30 +49,46 @@ NAN_METHOD(CopyablePersistent) {
   info.GetReturnValue().Set(New(p));
 }
 
+template<typename T> Global<T> passer(v8::Local<T> handle) {
+  return Global<T>(handle).Pass();
+}
+
+NAN_METHOD(PassGlobal) {
+  info.GetReturnValue().Set(passer(New(12)));
```
Comment (bnoordhuis) [inline test/cpp/persistent.cpp]: Clearly that should be 42.

**Comment by bnoordhuis** [comment]
LGTM

**Comment by kkoopa** [comment]
Just noticed that the commit message is wrong. But someting is weird with this test, if I move the added line somewhere else, some of the other tests fail on 0.10 and it is not evident why.

**Comment by kkoopa** [comment]
There, fixed the persistent tests.

**Comment by bnoordhuis** [comment]
Even more LGTM.

**Pull request merged**